assets/src/edit-story/components/panels/layer/useLayerSelection.js

```diff
@@ -58,14 +41,7 @@ function useLayerSelection(layer) {
 
       evt.preventDefault();
       evt.stopPropagation();
-      if (isBackground) {
-        // If background layer is clicked either select nothing or select only background element
-        if (backgroundHasElement) {
-          setSelectedElementsById({ elementIds: [elementInnerId] });
-        } else {
-          clearSelection();
-        }
-      } else if (evt.shiftKey && hasSelection) {
+      if (evt.shiftKey && hasSelection) {
```

Comment:
This seemed like an important logic. Looks like background element is not allowed to be in multi-selection? Did it move elsewhere?